build.sbt

```diff
@@ -58,18 +58,21 @@ lazy val scalatraSettings = Seq(
       case Some((2, _)) =>
         unusedOptions ++ Seq(
           "-Xsource:3",
-          "-release:8",
+          "-release:" + {
```

Comment:
I don't have much experience with SBT and those "virtual axes", so tell me if what I wrote doesn't make sense 😄 
What I was trying to do was to compile to Java 17 bytecode for the Jakarta build; otherwise stay with Java 8 as before.

Reply:
It has just occurred to me that I put this `-release` argument at the wrong location, under `unusedOptions` 🙄 
But looking at this question of the Java bytecode version, that made me think: shouldn't Scalatra simply be compiled for Java 8 for _all_ builds?
As the version requirement is actually a *runtime* requirement: the `javax` build requires 8+, while the `jakarta` build requires 17+ due to Jetty 12.
What do you think?

Reply:
If Java 17+ for Jetty 12 is the only runtime requirement as you mentioned, targeting Java 11 is better because backward compatibility is one of Scalatra's highest priority principals.

Reply:
See my remark in https://github.com/scalatra/scalatra/pull/1610#discussion_r1438942324, which proposes a solution where Scalatra 3.1.x could keep compatibility with Java 11 (`javax` build only, obviously).